### PR TITLE
[PHP 8.3] posix documentation updates

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: reference/posix/functions/posix-eaccess.xml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <type>bool</type><methodname>posix_eaccess</methodname>
+   <methodparam><type>string</type><parameter>filename</parameter></methodparam>
+   <methodparam choice="opt"><type>int</type><parameter>flags</parameter><initializer>0</initializer></methodparam>
+  </methodsynopsis>
+  &warn.undocumented.func;
```
Comment (Girgias) [inline reference/posix/functions/posix-eaccess.xml:15]:
```suggestion
```

File: reference/posix/functions/posix-eaccess.xml [hunk 2]
```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="utf-8"?>
+<refentry xml:id="function.posix-eaccess" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>posix_eaccess</refname>
+  <refpurpose>Description</refpurpose>
```
Comment (Girgias) [inline reference/posix/functions/posix-eaccess.xml:5]: Needs a description :)

File: reference/posix/functions/posix-fpathconf.xml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <type class="union"><type>int</type><type>false</type></type><methodname>posix_fpathconf</methodname>
+   <methodparam><type class="union"><type>resource</type><type>int</type></type><parameter>file_descriptor</parameter></methodparam>
+   <methodparam><type>int</type><parameter>name</parameter></methodparam>
+  </methodsynopsis>
+  &warn.undocumented.func;
```
Comment (Girgias) [inline reference/posix/functions/posix-fpathconf.xml:15]:
```suggestion
```

File: reference/posix/functions/posix-pathconf.xml [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <type class="union"><type>int</type><type>false</type></type><methodname>posix_pathconf</methodname>
+   <methodparam><type>string</type><parameter>path</parameter></methodparam>
+   <methodparam><type>int</type><parameter>name</parameter></methodparam>
+  </methodsynopsis>
+  &warn.undocumented.func;
```
Comment (Girgias) [inline reference/posix/functions/posix-pathconf.xml:15]:
```suggestion
```

File: reference/posix/functions/posix-sysconf.xml [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  <refpurpose>Returns system runtime information</refpurpose>
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <type>int</type><methodname>posix_sysconf</methodname>
+   <methodparam><type>int</type><parameter>conf_id</parameter></methodparam>
+  </methodsynopsis>
+  &warn.undocumented.func;
```
Comment (Girgias) [inline reference/posix/functions/posix-sysconf.xml:14]:
```suggestion
```

File: reference/posix/functions/posix-sysconf.xml [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+     </listitem>
+    </varlistentry>
+   </variablelist>
+  </para>
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Returns the numric value related to <parameter>conf_id</parameter>
```
Comment (Girgias) [inline reference/posix/functions/posix-sysconf.xml:41]:
```suggestion
   Returns the numeric value related to <parameter>conf_id</parameter>
```

**Review by Girgias — APPROVED** [review]
Minor nits but LGTM, thank you!

**Comment by devnexen** [comment]
Note to myself, documenting constants at some point.

**Pull request merged**